tests/shadowcasting_test.cpp

```diff
@@ -340,6 +340,77 @@ static void shadowcasting_float_quad(
     REQUIRE( passed );
 }
 
+static void do_3d_benchmark(
+    std::array<const float ( * )[MAPSIZE *SEEX][MAPSIZE *SEEY], OVERMAP_LAYERS> &transparency_caches,
+    const int iterations )
+{
+    float seen_squares[OVERMAP_LAYERS][MAPSIZE * SEEX][MAPSIZE * SEEY] = {{{0}}};
```

Comment:
```suggestion
    float seen_squares[OVERMAP_LAYERS][MAPSIZE * SEEX][MAPSIZE * SEEY] = {};
```
If it looks cleaner to you, you can just initialize these arrays like this and it should be equivalent (and maybe won't cause compiler complaints?).

Reply:
Looks good so far, I just hate the implicit-ness of it.